packages/genesys-spark-components/src/components/stable/gux-inline-alert/gux-inline-alert.tsx

```diff
@@ -3,11 +3,16 @@ import { buildI18nForComponent, GetI18nValue } from '../../../i18n';
 import { trackComponent } from '@utils/tracking/usage';
 import { GuxAlertAccent } from './gux-inline-alert.types';
 import translationResources from './i18n/en.json';
+import { hasSlot } from '@utils/dom/has-slot';
 
 /**
  * @slot - Slot for message.
  */
 
+/**
+ * @slot emphasis-text - Slot for emphasis text.
```

Comment:
This can be together with the default slot instead of documenting it twice.

 * @slot - Slot for message
 * @slot emphasis-text - Optional slot for emphasis text